specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/SessionPools.json

```diff
@@ -0,0 +1,737 @@
+          "x-ms-enum": {
+            "name": "SessionAuthStatus",
+            "modelAsString": true
+          }
+        },
+        "jwtToken": {
+          "description": "JWT token.",
+          "type": "string"
+        }
+      }
+    },
+    "CustomContainerTemplate": {
+      "description": "Custom container configuration.",
+      "type": "object",
+      "properties": {
+        "registryCredentials": {
+          "description": "Private container registry credentials for containers used by the Sessions of the Session Pool.",
+          "$ref": "#/definitions/SessionRegistryCredentials"
+        },
+        "containers": {
+          "description": "List of container definitions for the Sessions of the Session Pool.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SessionContainer"
+          },
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "ingress": {
+          "description": "SessionPool Ingress configuration.",
+          "$ref": "#/definitions/SessionIngress"
+        },
+        "sessionAuthConfiguration": {
+          "description": "Auth configuration of the Sessions in this Session Pool.",
+          "$ref": "#/definitions/SessionAuthConfiguration"
+        }
+      }
+    },
+    "SessionNetworkConfiguration": {
+      "description": "Session network configuration.",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "Network status for the Sessions.",
+          "type": "string",
+          "enum": [
+            "egressEnabled",
+            "egressDisabled"
+          ],
+          "x-ms-enum": {
+            "name": "SessionNetworkStatus",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "SessionPool": {
+      "description": "Container App Session Pool.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Container App Session Pool resource specific properties",
+          "type": "object",
+          "properties": {
+            "environmentId": {
+              "description": "Resource ID of the Session Pool's environment.",
+              "type": "string",
+              "format": "arm-id",
+              "x-ms-arm-id-details": {
+                "allowedResources": [
+                  {
+                    "type": "Microsoft.App/managedEnvironments"
+                  }
+                ]
+              },
+              "x-ms-mutability": [
+                "create",
+                "read"
+              ]
+            },
+            "containerType": {
+              "description": "The container type of the Sessions.",
+              "enum": [
+                "JupyterPython",
+                "FunctionsPython",
+                "CustomContainer"
```

Comment:
Are we using these, I think last time we discussed that we will use `python-lts` and `node-lts` ? We may need double confirm

Reply:
Thanks, missed this part, fixed now